ui/packages/app/web/src/components/MetricsGraph.tsx

```diff
@@ -149,7 +149,7 @@ export const MetricsTooltip = ({
   const highlightedNameLabel: Label.AsObject = nameLabel !== undefined ? (nameLabel) : ({ name: '', value: '' })
 
   return (
-    <div ref={setPopperElement} style={styles.popper} {...attributes.popper}>
+    <div ref={setPopperElement} style={styles.popper} {...attributes.popper} className="z-10">
```

Comment:
ah! So the tooltip needs a higher z-index? 
Cool! Thanks for fixing this in here too!